app/services/container_dashboard_service.rb

```diff
@@ -49,6 +50,13 @@ def image_metrics_data
     }.compact
   end
 
+  def refresh_status_data
+    {
+      :default_cred => @ems.present? && @ems.enabled ? 'Valid' : 'Invalid',
```

Comment:
```suggestion
      :default_cred => @ems.present? && @ems.enabled ? _('Valid') : _('Invalid'),
```

Reply:
@Fryguy we are not showing this text in ui, i need this as flag for condition to show the status or not.

Reply:
ohhh...interesting - maybe we should just make it a regular boolean then (and maybe change the key to `:default_cred_valid`, otherwise it's confusing.

Reply:
Actually, I'm also seeing `textual_authentications` used elsewhere, which I think _will_ be translated, so that makes me concerned this won't work on the other screens?

Reply:
@Fryguy All other provider dashboards like physical infra, cloud ... etc, has this field as  
`:default_cred => textual_authentications(@resource.first.authentication_for_summary)`- this gives result Valid/Invalid.I have written code in react for this( to show notification for !Valid cases), i want to use same code for container dashboard too, that is why i wrote a line like this .
:default_cred => @ems.present? && @ems.enabled ? 'Valid' : 'Invalid'.

Reply:
Is that Valid/Invalid from `textual_authentications` translated?

Reply:
@Fryguy im not sure, i don't see it in the function, so assuming it is coming from api.
 ```
def textual_authentications(authentications)
    return [{:label => _("Default Authentication"), :title => t = _("None"), :value => t}] if authentications.blank?

    authentications.collect do |auth|
      label = case auth[:authtype]
              when "default"           then _("Default")
              when "metrics"           then _("C & U Database")
              when "amqp"              then _("AMQP")
              when "console"           then _("VMRC Console")
              when "ipmi"              then _("IPMI")
              when "remote"            then _("Remote Login")
              when "smartstate_docker" then _("SmartState Docker")
              when "ws"                then _("Web Services")
              when "ssh_keypair"       then _("SSH Key Pair")
              else;                    _("<Unknown>")
              end

      {:label => _("%{label} Credentials") % {:label => label},
       :value => auth[:status] || _("None"),
       :title => auth[:status_details]}
    end
  end